task_12/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <title>Title</title>
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,300i,400,400i,500,500i,700,700i" rel="stylesheet">
+    <link rel="stylesheet" href="dist/css/style.css">
+    <script src="https://use.fontawesome.com/4a582f4e95.js"></script>
+    <script src="src/js/time.js"></script>
+</head>
+<body>
+<div class="wrapper">
+    <div class="content">
+        <header>
+            <div class="timeBlock">
+                <i class="fa fa-clock-o" aria-hidden="true"></i>
+                <time class="time">12:55:34</time>
+            </div>
+
+            <a class="logo" href="#">
+                <img class="logo__image" src="dist/img/pizza_logo.png" alt="pizza logo">
+            </a>
+
+            <button class="logInBlock" type="button">
+                <i class="fa fa-sign-in" aria-hidden="true"></i>
+                Login / Signup
+            </button>
+        </header>
+
+        <main>
+            <button class="button button_add">
+                <i class="fa fa-plus" aria-hidden="true"></i>
+                <span class="button_text">Add new pizza</span>
```

Comment:
For the all class names you use `camelCase` but only for a few `snake_case`. Why? It looks inconsistent too.

Reply:
it is not a `snake_case` - it`s BEM elements
For all class names i use `camelCase`